### ISSUE TITLE
Gradient exploding with aircraft dataset

### ISSUE THREAD

**Issue by greenlight2124** [visiondom/CHRF#1]
Hello, I tried the code with pre-trained weight, but the loss still became nan during training. Do you have any clue about this? Thanks so much.
